mungegithub/mungers/submit-queue.go

```diff
@@ -647,6 +657,34 @@ func (sq *SubmitQueue) getQueueStatus() []byte {
 	return sq.marshal(status)
 }
 
+// getMetrics returns a json representation of important stats about the
+// submit queue's health, for external monitoring/graphing/alerting.
+func (sq *SubmitQueue) getMetrics() []byte {
```

Comment:
Do not do this. We use Promethieus in Kubernetes for metric publishing. Search in the main repository, there should be lots of examples. :)